spec/lib/metasploit/framework/login_scanner/http_spec.rb

```diff
@@ -30,4 +30,16 @@
     end
   end
 
+  describe '#set_http_trace_proc' do
+    it 'configures http tracing' do
+      expect(subject.set_http_trace_proc(true)).to be_kind_of(Proc)
+    end
+  end
+
+  describe '#set_http_trace_proc' do
+    it 'does not log http tracing' do
+      expect(subject.set_http_trace_proc(false)).to be_nil
+    end
+  end
+
```

Comment:
These tests can probably also call the proc and check the output to be sure if we confident that output won't change.